--- a/pjlib/build/Makefile
+++ b/pjlib/build/Makefile
@@ -34,7 +34,7 @@
 #
 export PJLIB_SRCDIR = ../src/pj
 export PJLIB_OBJS += $(OS_OBJS) $(M_OBJS) $(CC_OBJS) $(HOST_OBJS) \
-	activesock.o array.o atomic_slist.o atomic_queue.o config.o ctype.o \
+	activesock.o array.o atomic_slist.o config.o ctype.o \
 	errno.o except.o \
 	fifobuf.o guid.o hash.o ip_helper_generic.o list.o lock.o log.o \
 	os_time_common.o os_info.o pool.o pool_buf.o pool_caching.o pool_dbg.o \
